scripts/main.js

```diff
@@ -71,4 +71,94 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+
+// Opdracht 1 - Gegevens aanspreken
+
+console.log('De afdeling Sales heeft ' + departments.sales.numberOfEmployees + ' medewerkers');
+console.log('Marketing is een leuke afdeling om te werken. ' +departments.marketing.description);
```

Comment:
Je kan in je code voor een betere leesbaarheid en consistentie nog zorgen dat je altijd een spatie plaatst voor en na de + bij het samenvoegen van strings.